src/lib/mina_state/snarked_ledger_state.ml

```diff
@@ -476,7 +476,7 @@ module Make_str (A : Wire_types.Concrete) = struct
 
     let all bs = Tick.(Run.run_checked (Boolean.all bs))
 
-    let ( || ) b1 b2 = Tick.(Run.run_checked Boolean.(b1 || b2))
+    let ( || ) b1 b2 = Tick.(Run.run_checked Boolean.(b1 ||| b2))
```

Comment:
This change is for disabling coverage for this file as it has compilation issue when building with `--instrument-with bisect_ppx`